research-hub-web/src/app/components/home/browse/browse.component.spec.ts

```diff
@@ -5,7 +5,9 @@ import { HttpClientTestingModule } from '@angular/common/http/testing';
 import { ApolloTestingModule } from 'apollo-angular/testing';
 import { CategoryCollection, AllCategoriesGQL } from '@graphql/schema';
 import { Observable, of } from 'rxjs';
-import { MockProvider } from 'ng-mocks';
```

Comment:
Looks like we have a mix of CSS fixes and test fixes?

Reply:
I refactored some things in a more angular-y way, and created new components. Those needed to be mocked in the old tests.

Reply:
All good